inst/stan/data/observation_model.stan

```diff
@@ -1,5 +1,7 @@
   int day_of_week[t - seeding_time]; // day of the week indicator (1 - 7)
   int model_type;                    // type of model: 0 = poisson otherwise negative binomial
+  int phi_mean;                      // Mean and sd of the normal prior for the
+  int phi_sd;                        // reporting process
```

Comment:
Shouldn't `phi_mean` and `phi_sd` be `real`, not `int`?

Reply:
Yes - haha this just flummoxed me a bit in testing